### PR TITLE
add evidencemodeler

### PR DESCRIPTION
**Pull request by rlibouba**
Hello, I'm opening this PR to add the EVidenceModeler tool. 
There's an error in the test that I can't correct. 
The error message indicates that the script tried to create a directory with an excessively long name, which failed. 
Thank you ! Have a good day!

### CODE REVIEW

File: tools/evidencemodeler/evidencemodeler.xml [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    <expand macro="requirements"/>
+
+    <command detect_errors="exit_code"><![CDATA[    
+    mkdir evm_dir_galaxy/ &&
+    cd evm_dir_galaxy/ &&
+
+    EVidenceModeler
+    --sample_id galaxy
+    --genome '$input_genome'
+    --gene_predictions '$input_predictions'
```
Comment (bgruening) [inline tools/evidencemodeler/evidencemodeler.xml:19]:
```suggestion

    ln -s '$input_genome' ./input_genome.fasta &&
    ln -s '$input_predictions' ./input_predictions.gff &&

    EVidenceModeler
    --sample_id galaxy
    --genome './input_genome.fasta'
    --gene_predictions './input_predictions.gff'
```

Can you please try something like that, also for the other files?

**Comment by bgruening** [comment]
let me know if you need help!

**Comment by rlibouba** [comment]
I've made some modifications. The current error indicates that the DB_file module is missing. It seems that this module is not present in the conda package. What do you think about this?

File: tools/evidencemodeler/test-data/galaxy.EVM.pep
Comment (bgruening) [inline tools/evidencemodeler/test-data/galaxy.EVM.pep:1]: is this file used?

Comment (rlibouba) [reply tools/evidencemodeler/test-data/galaxy.EVM.pep:1]: This file contains the protein sequences derived from the annotations of the predicted genes. These sequences can be used to validate predictions or for other analyses (homology, comparison between species, functional annotation). Only the input genome is in fasta format. The others are in .gff3 format. The output galaxy.EVM.pep provides a fasta output with protein sequences. I think this file could be useful, or maybe I'm wrong.

File: tools/evidencemodeler/evidencemodeler.xml [hunk 2]
```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0"?>
+<tool id="evidencemodeler" name="EVidenceModeler" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="21.05">
+    <description></description>
```
Comment (bgruening) [inline tools/evidencemodeler/evidencemodeler.xml:3]:
```suggestion
    <description>combines ab intio gene predictions, protein & transcript alignments into gene structures</description>
```

File: tools/evidencemodeler/evidencemodeler.xml [hunk 3]
```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0"?>
```
Comment (bgruening) [inline tools/evidencemodeler/evidencemodeler.xml:1]:
```suggestion
```

**Review by bgruening — APPROVED** [review]
Thanks @rlibouba!

I added a few more asserts, I hope that is ok. This looks great.

Btw. have you seen that the developers of this tool do not plan to maintain it and recommend another tool instead, called Ginger?

https://github.com/EVidenceModeler/EVidenceModeler/wiki#alternatives-to-evm

**Comment by rlibouba** [comment]
Thanks @bgruening !
I hadn't seen that information. I'll take a look at this other tool

**Pull request merged**